server/base/src/main/java/org/apache/accumulo/server/util/fateCommand/FateTxnDetails.java

```diff
@@ -73,8 +73,8 @@ public FateTxnDetails(final long reportTime, final AdminUtil.TransactionStatus t
     if (txnStatus.getTop() != null) {
       step = txnStatus.getTop();
     }
-    if (txnStatus.getTxName() != null) {
-      txName = txnStatus.getTxName().name();
+    if (txnStatus.getfateOp() != null) {
+      fateOp= txnStatus.getfateOp().name();
```

Comment:
Run the maven command:
`mvn clean package -DskipTests`
to quickly format the project